src/agent/config.rs

```diff
@@ -32,6 +32,8 @@ pub struct Config {
     pub metrics_server:        metrics::Config,
     #[serde(default)]
     pub remote_keypair_loader: services::keypairs::Config,
+    #[serde(default)]
+    pub opentelemetry:         OpenTelemetryConfig,
```

Comment:
Let's make this one an option and actually not construct it if it's not given. I know that it's probably harmless but it's better to be clear.